src/ConstructiveSolidGeometry/Intervals.jl

```diff
@@ -0,0 +1,8 @@
+@inline _left(r::T) where {T <: Real} = -r
```

Comment:
Shouldn't this be 0?

Reply:
also, `_left(r::T)` in general is not good. It depends on what `r` is. Like it's different for `x` and `r`

Reply:
I only used `_left` for intervals that were not the cylindrical `r`.
(I can see that using `r` as variable name is confusing... 😅)

Reply:
I guessed so, but non the less the name `_left` for a function is really unclear. It should somehow give the reader of the code an idea what it will do. When passing an interval it is clear, but not when passing a number. The computer also can't tell whether the single number passed is an radial coordinate or an linear one. 
Here, I don't the see the point to make the name of the function as short as possible. þ

Reply:
Okay, we will go for `_left_linear_interval`.